index.js

```diff
@@ -79,8 +83,9 @@ app.post('/v1/autodock', (req, res) => {
     })
     form.on('fileBegin', function (name, file){
         if (fs.existsSync(uploadDirectory + name + '.pdbqt')) {
-            res.status(400);
             errorMsg = 'Multiple files with same name uploaded?';
+            res.status(400)
+            return res.send(errorMsg)
```

Comment:
Same as above

Reply:
In here is where it needs to be fixed, you are correct.